sound/hybrid_font.h

```diff
@@ -145,14 +147,32 @@ class HybridFont : public SaberBase {
           swing_player_->set_fade_time(swing_player_->length() - swing_player_->pos());
           swing_player_->FadeAndStop();
           swing_player_.Free();
-          swing_player_ = PlayPolyphonic(&swng);
         }
       }
       else if (!swing_player_) {
-        swing_player_ = PlayPolyphonic(&swng);
+        if (!swinging_) {
+          swing_player_ = PlayPolyphonic(&swng);
+        }
       }
     } else {
-      PlayMonophonic(&swing, &hum);
+      if (!swinging_) {
+        PlayMonophonic(&swing, &hum);
+      }
+    }
+  }
+  
+  void StartSlash() override {
+    if (swing_player_) {
+      // avoid overlapping swings, based on value set in ProffieOSSwingOverlap.  Value is
+      // between 0 (full overlap) and 1.0 (no overlap)
+      if (swing_player_->pos() / swing_player_->length() >= config_.ProffieOSSwingOverlap) {
```

Comment:
So how is this going to work?
By the time we get here, we will have almost always started a swing very recently.
Because of inertia, we'll start at a speed near zero, and once we reach 200 we'll start a swing, and then we'll get here soon after, and then we won't play a slash because we just started a swing.

I have two possible ideas for how we deal with this:
1) If a swing is playing, we transition to slash regardless of overlap. (However, if a *slash* is playing, we take the overlap into account.)
2) Instead of looking at the swing speed to decide if it's a slash or a swing, we could look at the derivative.   When we reach the swing speed, if the speed is accelerating rapidly, we could just start a slash right away instead of playing a swing first.

Reply:
I agree that it seems counter intuitive based on the code but in testing this works decently.  I think your idea of transitioning to slash regardless of overlap will sound better.  I'll give that a good.

Reply:
I actually think using the acceleration would sound the best, I just don't know if it would *work* the best.

Reply:
Using an acceleration change derived from speed, sampled every 10 ms